src/lib/services/CommandLineSupport.ts

```diff
@@ -8,6 +8,7 @@ import {SpinnerManager, NoOpSpinnerManager} from './SpinnerManager';
 export type ResultHandlerArgs = {
 	code: number;
 	isSuccess: boolean;
+	hasResults: boolean;
```

Comment:
What is this property used for? I can see it being set at line 105, but I can't find any uses of it.

Reply:
Thanks for catching this. My newer solution doesn't need the property anymore and we are including the contents of stdout anytime the script execution is rejected.